i18n/fr.po

```diff
@@ -262,5 +262,8 @@ msgid ""
 "character, one number, and one special character."
 msgstr ""
 "Le mot de passe doit comporter au minimum {{minPasswordLength}} et au maximum {{maxPasswordLength}} "
-"caractères, dont au minimum un caractère miniscule, un caractère majuscule "
-"un chiffre, et un symbole."
\ No newline at end of file
+"caractères, dont au minimum un caractère miniscule, un caractère majuscule, "
+"un chiffre, et un symbole."
+
+msgid "Invalid password"
+msgstr "Mot de passe invalide"
```

Comment:
this is just added for testing purposes to confirm the translations get updated as you type after changing languages